.github/workflows/lint_python.yml

```diff
@@ -7,10 +7,11 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
       - run: pip install --upgrade pip wheel
-      - run: pip install bandit codespell darker flake8 flake8-2020 flake8-bugbear
+      - run: pip install bandit black codespell darker flake8 flake8-2020 flake8-bugbear
                          flake8-comprehensions isort mypy pytest pyupgrade safety
       - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
-      - run: darker --check .
+      - run: black --check .
+      - run: darker --check . || true  # Fails in a GitHub Action
```

Comment:
The workflow should fail if there is any non-Black formatting on modified lines. So ` || true` would need to be removed. But as said earlier, `pytest-darker` already runs `darker` in [.github/workflows/python-package.yml](https://github.com/akaihola/darker/blob/8a1835c952b9743ea5cb6d40d23eb9c6458e7828/.github/workflows/python-package.yml).